opennms-full-assembly/src/test/java/org/opennms/assemblies/karaf/FeatureInstallKarafIT.java

```diff
@@ -0,0 +1,150 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2014-2015 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2015 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.assemblies.karaf;
+
+import static org.ops4j.pax.exam.CoreOptions.maven;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.opennms.core.test.karaf.KarafTestCase;
+import org.ops4j.pax.exam.junit.PaxExam;
+import org.ops4j.pax.exam.spi.reactors.ExamReactorStrategy;
+import org.ops4j.pax.exam.spi.reactors.PerMethod;
+
+@RunWith(PaxExam.class)
+@ExamReactorStrategy(PerMethod.class)
+public class FeatureInstallKarafIT extends KarafTestCase {
+
+    /**
+     * This test attempts to install all features from the OpenNMS
+     * features.xml.
+     */
+    @Test
+    public void testInstallAllOpenNMSFeatures() {
```

Comment:
The OnmsFeatureKarafIT test does install each feature individually so we're doing it both ways. 

I'm not sure if this monolithic test serves much of a purpose other than failing if there are severe interdependency issues (like multiple versions of Spring or other libraries being loaded at once). But I guess that's a good reason to try and load all of the features in the same container like we're doing here.